src/components/organisms/TimingDeleteModal/TimingDeleteModal.scss

```diff
@@ -0,0 +1,14 @@
+@import "scss/constants.scss";
+
+.form {
```

Comment:
This should probably be encapsulated inside a component class, like
```suggestion
TimingDeleteModal {
  .form {
```

Reply:
done 917110d